lib/univalue.cpp

```diff
@@ -163,9 +163,9 @@ bool UniValue::findKey(const std::string& key, size_t& retIdx) const
 
 bool UniValue::checkObject(const std::map<std::string,UniValue::VType>& t)
 {
+    size_t idx = 0;
     for (std::map<std::string,UniValue::VType>::const_iterator it = t.begin();
          it != t.end(); ++it) {
-        size_t idx = 0;
```

Comment:
Comments:
1) Please put into a separate pull request, as this is unrelated to documentation changes.

2) Let the compiler perform the hoisting.  No need to do it manually.

General rule:  Limit scope of a variable, and let the compiler figure out the rest.   If it becomes a performance issue notable on profiles, revisit.